src/js/apps/patients/view/view_app.js

```diff
@@ -54,4 +59,20 @@ export default App.extend({
 
     return groups;
   },
+  _filtersById(viewId, currentClinician) {
+    const clinician = currentClinician.id;
+
+    const filters = {
+      'owned-by-me': { clinician },
+      'actions-for-coordinators': { role: 'coordinator' },
+      'new-actions': { created: moment().subtract(1, 'day').format() },
+      'updated-past-three-days': { updated: moment().subtract(3, 'days').format() },
+      'done-last-thirty-days': {
+        updated: moment().subtract(30, 'days').format(),
```

Comment:
wasn't sure one the time situation here...does this need to be `.utc()`'d?

Reply:
nope.  The time is relative to the user so as long as it has a timestamp (which it will here), we're good.